pkgs/tools/misc/lolcat-rs/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchgit, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lolcat-rs";
+  version = "1.5.0";
+
+  src = fetchgit {
```

Comment:
This should use fetchFromGitHub

Reply:
I tried that first, but it failed because the upstream author didn't create any releases or tarballs.

Reply:
I think in this specific case `fetchCrate` may make more sense, because the upstream doesn't seem to tag releases on GitHub.

```nix
  src = fetchCrate {
    crateName = "lolcat";
    inherit version;
    hash = "sha256-RVYmBfF9BlS9WSBcnptlgZ2nF7b7NC+58cZMHVX7SxQ=";
  };
```